DependencyInjection/EzSystemsEzSupportToolsExtension.php

```diff
@@ -15,6 +15,16 @@
 
 class EzSystemsEzSupportToolsExtension extends Extension
 {
+    public function getAlias()
+    {
+        return 'ez_support_tools';
```

Comment:
We made arbitrary choices of extension names in the past. Looking back at this, I'm not sure if those were the right choices. The extension is meant for eZ Platform (and its variations) only ATM, so I feel like we should indicate that by
`ezplatform_support_tools` or `ezplatform_system_info`.

Not insisting though, since this would be a fallback given by Symfony if `getAlias()` was not defined (AFAIR).